docs/configuration.md

```diff
@@ -96,9 +96,13 @@ The name of the cluster (to be referenced later from the data cube).
 
 The database type of the cluster. Currently only Druid is supported.
 
-**host** (string)
+**url** (string)
+
+The url address (protocol:hostname:port) of the cluster. If not protocol, http is assumed. If no port, 80 is assumed for plain http, and 443 for secure https.
```

Comment:
I would keep protocol mandatory.
What do you think about the following example/pattern http[s]://hostname[:port]/.

Reply:
That's bad writing on my part. "http" is assumed in old `host` property. Here, protocol is mandatory. To be exact, value must be valid "url" as understood by `URL` class from node/browser.